### PR TITLE
APIS-6490: copy command logic from APM

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**

### CODE REVIEW

File: app/uk/gov/hmrc/apiplatform/modules/developers/domain/models/LoggedInState.scala [hunk 1]
```diff
@@ -18,17 +18,12 @@ package uk.gov.hmrc.apiplatform.modules.developers.domain.models
 
 import enumeratum.{EnumEntry, PlayEnum}
 
-sealed trait LoggedInState extends EnumEntry {
-  def isLoggedIn: Boolean                = this == LoggedInState.LOGGED_IN
-  def isPartLoggedInEnablingMFA: Boolean = this == LoggedInState.PART_LOGGED_IN_ENABLING_MFA
-}
+sealed trait LoggedInState extends EnumEntry
```
Comment (AndySpaven) [inline app/uk/gov/hmrc/apiplatform/modules/developers/domain/models/LoggedInState.scala]: Kill all enums :-)

File: run_all_tests_no_coverage.sh [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 export SBT_OPTS="-XX:+CMSClassUnloadingEnabled -XX:MaxMetaspaceSize=1G"
-sbt clean compile test it:test
+sbt run-all-tests
```
Comment (AndySpaven) [inline run_all_tests_no_coverage.sh]: Probably delete this file.  They are almost never used by anyone

File: app/uk/gov/hmrc/thirdpartyorchestrator/commands/applications/domain/models/DispatchSuccessResult.scala [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+package uk.gov.hmrc.thirdpartyorchestrator.commands.applications.domain.models
+
+import play.api.libs.json.Json
+
+import uk.gov.hmrc.apiplatform.modules.applications.core.domain.models.ApplicationResponse
+
+case class DispatchSuccessResult(applicationResponse: ApplicationResponse)
+
+object DispatchSuccessResult {
```
Comment (AndySpaven) [inline app/uk/gov/hmrc/thirdpartyorchestrator/commands/applications/domain/models/DispatchSuccessResult.scala:25]: Probably need to sort what TPA returns in that most LDT are as millis except TPOUA.

File: run_all_tests_no_coverage.sh (hunk 2 above)
Comment (mattclark-zerogravit) [reply run_all_tests_no_coverage.sh]: will do

File: app/uk/gov/hmrc/apiplatform/modules/developers/domain/models/LoggedInState.scala (hunk 1 above)
Comment (mattclark-zerogravit) [reply app/uk/gov/hmrc/apiplatform/modules/developers/domain/models/LoggedInState.scala]: done

**Review by Tobias1087 — APPROVED** [review]

**Pull request merged**